src/RunCommand.php

```diff
@@ -77,14 +77,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $progress->finish();
 
             if ($pictures) {
-                error_log(sprintf("Uploaded/refreshed %d of %d image file(s)", $pictures[0], $pictures[1]));
+                error_log(sprintf(PHP_EOL."Uploaded/refreshed %d of %d image file(s)", $pictures[0], $pictures[1]));
             }
         } else {
             unset($this->config['phonebook']['imagepath']);             // otherwise convert will set wrong links
         }
 
         // fritzbox format
-        $xml = export($filtered, $this->config);
+        $XMLPhonebook = export($filtered, $this->config);
```

Comment:
why rename here? Also upper case variable names are uncommon. please revert.

Reply:
OK Sorry for length...
My reasons where:
1. Back in the Eighties/Nineties we named C/C++ variables like so: "iCounter" or "strName" because we had shitty IDEs that could not reflect the type of a variable and type was important. 
But nowadays we have shiny PHPStorms etc. that in many cases tell you the type of the var. BUT the most important part is not the type but the "Why & What" is stored in there. So to me it seemed inferior to give the type "xml" instead of what's really in there: a phone book.
OK PHP is weak typed - so maybe it makes sense to add the type also. Because in case of XML we have two type flavors: SimpleXMLElement & String. 
So, thats why I think the "Phonebook" thing is important, the type XML / XMLStr comes second. But also OK. Not all IDEs can infer types in PHP.
2. In the 80s/90s we used short var names because there was no intellisense - these times are gone. Long names are fast now.
3. A short var name need often explanation in a comment. "What is stored here? And why?" - so longer names save comments. A good thing.

I would vote for keep the long name.

Reply:
fine with me, but then please `xmlPhonebook`. Could you double check if we have xml phonebooks anywhere else and also rename that one to keep it consistent?